hack/lib/build/binaries.sh

```diff
@@ -225,11 +225,22 @@ os::build::internal::build_binaries() {
         os::util::ensure::gopath_binary_exists 'goversioninfo'
         local major="${OS_GIT_MAJOR}"
         local minor="${OS_GIT_MINOR%+}"
+        local patch="${OS_GIT_PATCH}"
         local windows_versioninfo_file=`mktemp --suffix=".versioninfo.json"`
         cat <<EOF >"${windows_versioninfo_file}"
 {
 	"FixedFileInfo":
 	{
+		"FileVersion": {
+			"Major": ${major},
+			"Minor": ${minor},
+			"Patch": ${patch}
```

Comment:
We could add the SHA as the build id in here if you wanted, but I have no idea if that is appropriate.